src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -0,0 +1,85 @@
+from copy import deepcopy
+from dataclasses import dataclass
+from functools import lru_cache
+from types import MappingProxyType
+from warnings import warn
+
+import os
+import torch
+import torch.overrides
+
+from torch.fx import GraphModule
+
+from openvino.frontend import FrontEndManager
+from openvino.frontend.pytorch.fx_decoder import TorchFXPythonDecoder
+from openvino.runtime import Core, Type, PartialShape, serialize
+
+from typing import Callable, Optional
+
+
+import numpy as np
+def openvino_compile(gm: GraphModule, *args, model_hash_str : str = None):
+    core = Core()
+
+    device = 'CPU'
+
+    if os.getenv("OPENVINO_DEVICE") is not None:
+        device = os.getenv("OPENVINO_DEVICE")
```

Comment:
Can we make this a parameter of torch.compile backend instead of passing via environment variable? Passing arguments via env vars can be used as a workaround, usually we always have good enough place in API for such parameters. Just compare with ov.compile_model, we are not using env vars to pass device time, so I would propose not to use env vars in this case.

Reply:
As discussed, will implement this after pre-release